### PR TITLE
Unhardcoded decorations

### PR DESCRIPTION
**Pull request by Dzierzan**
Decorations like trees or stones can be now placed anywhere, regardless the tileset.

image

### CODE REVIEW

File: mods/hv/bits/shadow-decoration.pal [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+0 0 0 45
+0 0 0 40
+0 0 0 35
+0 0 0 30
+0 0 0 25
+0 0 0 10
+0 0 0 0
+0 0 0 200
+0 0 0 180
+0 0 0 170
```
Comment (Mailaender) [inline mods/hv/bits/shadow-decoration.pal:22]: This is a duplicate of line 6

File: mods/hv/bits/shadow-decoration.pal [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+0 0 0 80
+0 0 0 65
+0 0 0 45
+0 0 0 40
+0 0 0 35
+0 0 0 30
+0 0 0 25
+0 0 0 10
+0 0 0 0
+0 255 0 255
```
Comment (Mailaender) [inline mods/hv/bits/shadow-decoration.pal:132]: This seems far worse than the all shades of grey palette. https://raw.githubusercontent.com/OpenHV/OpenHV/712389a3b019fb02ce2589c64677c28675fd11fb/mods/hv/bits/shadows.pal

**Comment by Dzierzan** [comment]
image

This is how I've done it. So I could just apply the palette and have the correct shading. Hence duplicates.

**Comment by Mailaender** [comment]
I see. You edited the palette instead of the sprites. Smart.

**Pull request merged**